pypeit/reduce.py

```diff
@@ -1097,6 +1140,17 @@ def local_skysub_extract(self, global_sky, sobjs, spat_pix=None, model_noise=Tru
         # Set initially to sciivar in case no obects were found.
         self.ivarmodel = np.copy(self.sciImg.ivar)
 
+        # Add here slitmask stuff: assign RA,DEC, OBJNAME to detected objects and add undetected objects
```

Comment:
I'm not sure why this is being done here, but in my opinion it should not be. This code works with your final set of objects which should be determined upstream.